docs/general/images/architecture/client-km-compute.svg

```diff
@@ -1,4 +1,114 @@
re5sfWWx2BWgag8bg7PXnprsfKaGyepsrkm1cQV2IaKIkMcqSdCLpIOvmkuZofoEyFZsoX7eGmPJTPFXDJQ1PpuDsny7U6dqZlLh7exlpefYTvFqpNO5do0JGS6tdlGyJWxNV0qQBDvgjmos9b6MOK0bvFbabE+EQxwd9sbUh75Nq1G83gYh6FxUBUyOfYVc84V+dpjijVqvqqRSFWSGgUThbHVetFEFptOnXxcvGalFBRqWwYgyDtp+oEcdyhbVXKxULhVzzlRlO87b4h9v96Z111Rc1zyRD9syNF2Ok8H5sPjYdiwmf7drzsZcI+8Vfxo7F3oo5NHk/5Pe75ZOx5dhWTAfUytopeVmRd/usdg1NmLtYehj91a5j7KfBhPjP1MgvK8aS1/McG9n4ylwr4dWyh9prJfEnEcwcIs2r0Ss3ACXkl1DVVwKKSCKGIw9699ckiTto301oTC5SX8enHd5LkhiCEggPK19I82yX/GS9KUDcboXWtwuMXAdE3qjgjEsNhsaPxHGbBT3RhWmKSePbs25ka4zcAiAC9tHD9iw6nJ9BQhGKnDT/hBpo/85p8ocztlnL1r1jWA+wYcKW/god5vFV5M8RJemNK42dQkTjWmgDNSSjuKicrFYV7WrFo+DpsHbrD47l/nQmh5Kw864Rdv/UwE+fQGlQ8/qWrV6ryUyWuo6JK1yuXC2H8RUdHMxpbipKOOzzJkIxnssJ53rtGREOpK9xl0Guf6duiu70WfDQniXCuLf89C7gaezs2D8kLHcm9En8ZXm4Nz5QfP8NVZAVaiEFApyOOoa1Rp/6x4lNUJbtTqcNrjfggteTc7umjDpfhwEPa8FOsseoOVGjPaO0Amsn6On3qg74legm3jyrz75vWef/RK7dk+avlgvOf2QecOPV+7hqXe+LvKfD7C+q4PfAkNCU8PZHKO/G7fX8+2SH/qFnk8aIR0rXSec7d0IoMpId5v8HAAHAoOLvElf/uPfI/t4+WgAAnHjhg/XhoHePn/WVol1KTyMANCigIvkXachSJ/zFNwWq9detoyM43l3TALYWW6fN+e5ebCv81S4DtXn/PLLIhvD6OFP0ynsu4KPOHHCiR7eI7CwMzxR1KD7qRiv75MO97G7s8AHYyrwAb22+hu+odeCLNn6vT9oqrPPjI9y9XYAvMvvwJO/WHw4HeKNCAd2d5REQ5XdFiL9PW0M+X497+WZOw94EVA+FgvYf+KFM4tX5mej1wdqXGPscqlt1b8EP3d7NvftafPlfyyLy2X4Ya9/gP30YW9vtxJXjf5XiTp3NPQv4MtKpX+G78iUgDGahXEydpQvCwOwUn9a5bA2uBhmMb/NFYwQbjemZ1wF0KCq8hfzuRnzSQste4061OOGu7mmzB0nS+zQuClgdHmifiuJO0QOm89ADcBJg60mNBQFAbah1va6srdoHyq2L/1y906l6Fabzr8pRCV7kVMtVsO4bdlo0/Kcp9d6RduOvFjRCZDXZS/rQyJDepC/pSXpR7PHCH/S1V1k5Gu138VcbX7/uY2BdkGzT2lHAFXOafhAfrKy2GRboTL2RtjQyJJn0p75Jd9KJxocmihMW+I50YzpFT+mDaPiiS+MxMHhjEbfPRB8pnp658Vjx5kz0ruKlNjFkcKvY9V+JnpD9IIor9kUibCYfgyjS8DHfoLI5Q66DkNV2bAYB0OEHOZiNAQKv6LhtItqLOiPmKeujrUQgkWUlClryVIlGIRlXYmAj+SUWJRgIuFs5tGWgNV/fjRCo8RJKBRCwuVQIA0KlIkh4YoSvZZ3416x7SgHosaBUDAEzSiVIhFwqhYSSUhksSOFyWKaSDBMgQfCb/y0BYERLC8hQKwpIiQtasCyraHumhLItLlPu1YRhOweUXpJcpct5amFoqoYCm2bWpsVj0EGIGWYtPy4qplg7OG2ZxfLWDJswaNq/KKtgkzAwzRzutyLoqIuSZcIFYoHC+SWlznM4t+YYUxzFiC6Xsmbignhti2t5nudDb4FIlLk4rVapfNNZE2QKg5ZaBDdZW66WiU62/Kqo9dvoNWeBd3VTle/6gmtxyYtVXFXGcHojYSTjGS1RLILMCxSj1YLu7Weuad1ajLGZFUUgaP5CyBzTMgjdL/wwo4Pf8SuXzP/hMwA4dFDREVVbCNIzHYBxwFhjL/aZBBGHLIKK4ziJ07hOi8ETiCRya0ShUBMj4iy9P4ZKCqoYYKWJoR62HcMZY4THF+TijjGRWCKVyRVKlVqj1ekNRpPZYrXZHU6X2+P1QWFwBDkFJRW1LRpaOnrbSAZGJmYWVjZ2Dk4ubh5ePn4BlKCQsIioWKfjl5Ds6UhJ1xnPRMvKySsoYpSUVVTV7KhraNp1XqUq1WrUoqlTr0GjJs1aHNSqTbsOnbp0O+SwHr369BtwxCARdAxDho0YdRTTmGOOG8fCNmHSlGkzZs3hmLdg0ZJlK1atWce1YVOjJs1atGo7v2bsID6/PBAgROq2khQTWSBwpHuMHkuSLc+ydCirwJX2dZGvYxDmnVt3+I8z7CpU7rNxWSOF/fJSHCZuXspAT6iDjHeShfjhMmHUJ6EfTSBYzUfLhpuNZSMTl5IYywpcGdQah0Rq4qabqXNe+MPkEw2k04HOrHQzlyukHrIhBfnsHKVbShQ0JRHUpm6v0EWvOPJlh9I3uTCLOokUlrbPrNykEg8j9KIGbLBanC/p+a+dDxd+/snm8+Lv/p2z+YR3eysucKGS5vXrXTFB4duVPq2K3Vs+3czp5hUVhQAhKE7S2jBA5VU0IHNDS0UDyFPFJK31dpzg2nBIHJmDiyJ4GEEx6V4sxIiSBUZHRZM6JihGF8s8gtLEPUIXjhBOwAgP3bBrdHXjtpUd0vaBYm2ROMUJ37+K3fzBnAN1PPWVwk0+lA7/p1+BeOogfYsHT2wvfX/m5vtj7jFG08mju7eRv2LguOej5sXhvzCz+SMy69Lc+6bmA+fnAuYLAAAA") format('woff2');
```

Comment:
I would uncheck "Embed Fonts" in draw.io svg export. It added 800KB of font data and pre-rendered fallback images into a 30KB svg.

When unchecked it creates a 30KB svg that loads +50KB (latin-only) from google fonts. Drawback is possible google tracking.

We can avoid tracking by self-hosting and manually editing the svg :shrug:  or use a boring web-safe font

Reply:
hm it could reuse font if we inline `<svg...` instead of `<img src=`

Reply:
I can't seem to spot a difference. Tahoma should be sans-serif right?

<img width="2183" height="1339" alt="image" src="https://github.com/user-attachments/assets/ee6a7b7b-6777-4fa6-8909-3c5be58cb033" />

Reply:
talked with Kaja: it's annoying to change fonts in each element. Draw.io does not seem to allow grouping all elements and inheriting a font from one place, specifically on texts above arrows - they are foreignObject html

Reply:
oh, ctrl+a (select all) + set custom font "Tahoma, Verdana, Segoe, sans-serif" works